packages/cursorless-vscode/package.json

```diff
@@ -20,7 +20,7 @@
       "description": "Tests"
     }
   ],
-  "version": "0.29.0",
+  "version": "1.0.0",
```

Comment:
I think this should be enough for the new version to be used.

Reply:
Not sure have a look at the last release should be the same

Reply:
Yeah that was all